creator-node/src/routes/tracks.js

```diff
@@ -210,9 +210,14 @@ module.exports = function (app) {
       } = req.body
 
       // Input validation
-      if (!blockchainTrackId || !blockNumber || !metadataFileUUID) {
+      if (
+        !blockchainTrackId ||
+        !blockNumber ||
+        !metadataFileUUID ||
+        !transcodedTrackUUID
+      ) {
         return errorResponseBadRequest(
-          'Must include blockchainTrackId, blockNumber, and metadataFileUUID.'
+          'Must include blockchainTrackId, blockNumber, metadataFileUUID, and transcodedTrackUUID.'
```

Comment:
@raymondjacobson this is the one change i wanted to flag to you, detailed in PR description